app/views/categories/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="top-links">
+  <%= link_to 'BACK', session.delete(:return_to) || root_path, class: "back-button" %>
+  <h2 class="category-h">Categories</h2>
+  <%= button_to 'Sign Out', destroy_user_session_path, method: :delete, class: 'sign-out-button' %>
+</div>
+
+<div class="section-container">
+  <section class="category-section">
+    <% @categories.each do |category| %>
+      <a class="category" href=<%= category_transactions_path(category) %>>
+        <div class="img_container">
+          <img src=<%= category.icon %> alt=<%= category.name %> />
+        </div>
+        <div class="category-info">
+          <h3><%= category.name %></h3>
+          <p>Total Amount: <%= category.transactions.sum(:amount) %></p>
+        </div>
+      </a>
+    <% end %>
+  </section>
+</div>
+
+<div class="new-category-button">
+  <%= link_to 'Add a New Category', new_category_path, class: "new_category" %>
+</div>
```

Comment:
- [ ] [REQUIRED] Great functionalities I must commend your efforts @Itswali 👍🏽 , however, we need to do some UI fixes here. The "Add New Category" button should also be at the bottom of the screen and also ensure that the same color on the navbar are buttons are applied to your design as used in the Behance design.
- [ ] Also, take a look at the way each item of a category is been displayed flex such that the image, name, and total amount of the category stays on a single row but three columns.
- [ ] Since this is the landing page. It will be professional to remove the "Back" button on it.
- These are hard project requirements so let's implement the fixes 👍🏽 

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="300" alt="Screenshot 2023-07-05 at 22 43 02" src="https://github.com/Itswali/transaction-app/assets/58692590/00aa5131-6df5-44c6-a9ee-e2f1b3a7173c"> | <img width="388" alt="Screenshot 2023-07-05 at 22 39 37" src="https://github.com/Itswali/transaction-app/assets/58692590/b25609f6-7344-4bc8-b92f-19554d6df37c">

Reply:
Thank you for suggestion I updated the button color and position as well as navbar color